api/swagger.yaml

```diff
@@ -2802,10 +2822,19 @@ definitions:
         additionalProperties:
           type: "string"
       Data:
-        description: "Base64-url-safe-encoded secret data"
+        description: |
+          Base64-url-safe-encoded ([RFC 4648](https://tools.ietf.org/html/rfc4648#section-3.2))
+          data to store as secret.
+
+          This field is only used to _create_ a secret, and is not returned by
+          other endpoints.
```

Comment:
Hmm, we should probably have two different types then. One without this field. Since this already exited we can wait for another PR